src/chains/ethereum/ethereum/src/api.ts

```diff
@@ -1402,15 +1402,14 @@ export default class EthereumApi implements Api {
    * 2: `DATA`, 32 Bytes - the seed hash used for the DAG.
    * 3: `DATA`, 32 Bytes - the boundary condition ("target"), 2^256 / difficulty.
    *
-   * @param filterId - A filter id.
    * @returns The hash of the current block, the seedHash, and the boundary condition to be met ("target").
    * @example
    * ```javascript
-   * console.log(await provider.send("eth_getWork", ["0x0"] ));
+   * console.log(await provider.send("eth_getWork", [] ));
    * ```
    */
   @assertArgLength(1)
```

Comment:
I was a bit early to this approval. Please update this line to:

`@assertArgLength(0)`